packages/fast-jss-utilities/src/colors.ts

```diff
@@ -0,0 +1,51 @@
+import { adjustContrast, contrast, ContrastFunction, ensureContrast } from "@microsoft/fast-colors";
+import { clamp, memoize } from "lodash-es";
+
+/**
+ * Hashing function for contrast memoization
+ */
+export function contrastHasher(...args: Array<string | number>): string {
+    return args.map((value: string | number) => {
+        return typeof value === "string" ? value.toLowerCase() : value;
+    }).join("");
+}
+
+const memoizedContrast: ContrastFunction = memoize(contrast, contrastHasher);
+const memoizedEnsureContrast: ContrastFunction = memoize(ensureContrast, contrastHasher);
+const memoizedAdjustContrast: ContrastFunction = memoize(adjustContrast, contrastHasher);
+
+/**
+ * Export memoized contrast functions to prevent calculating the same color
+ * multiple times
+ */
+export { memoizedContrast as contrast };
+export { memoizedEnsureContrast as ensureContrast };
+export { memoizedAdjustContrast as adjustContrast };
+
+/**
+ * Contrast ratios for normal and large UI elements and text.
+ * See https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
+ * for more information
+ */
+export enum WCAGAAContrastRatios {
+    /**
+     * The contrast ratio for normal size text elements and UI elements less than 3px
+     */
+    normal = 4.5,
+
+    /**
+     * The contrast ratio for large size text elements and UI elements 3px or greater
+     */
+    large = 3
```

Comment:
i find this a little unintuitive, because I would expect `WCAGAAContrastRatios.normal` to be a standard ratio and `WCAGAAContrastRatios.large` to be a larger ratio, not relating to the size of the UI elements.

Reply:
Do you have a suggestion to add clarity? This makes sense to me because a larger element would require a lower contrast ratio because the element is physically more prominent and noticeable.

Reply:
maybe `default` or `normal` and `low`?

Reply:
Who is to say what is 'normal' or 'low' as this may vary depending on many accessibility factors.  I would consider sticking with sizes.  'small' 'medium' 'large' and we can extend a few others later if needed. 'extra-large' 'extra-small' for example.

Reply:
Leaving as is but clarifying the enum name.